src/styles/main.scss

```diff
@@ -1,3 +1,159 @@
+@import "stars";
+@import "header";
+
+$buy--color: #00acdc;
+$text--color--one: #060b35;
+$text--color--two: #616070;
+$header--bcg--color: #fff;
+$nav__item--color: #000;
+$card__hover: #34568b;
+$border--cllor: #f3f3f3;
+$column-count: 1;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text--color--one;
+}
+
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 50px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px;
+  transition: 1s;
+
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    .main {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    .main {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    .main {
+      grid-template-columns: repeat(4, 200px);
+    }
+}
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  border: 1px solid $border--cllor;
+  background: $header--bcg--color;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__info {
+    color: $text--color--one;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: auto 0 4px;
+
+    &:hover {
+      color: $card__hover;
+    }
+  }
+
+  &__code {
+    font-size: 14px;
+    line-height: 14px;
+    margin: auto 0 16px;
+    color: $text--color--two;
+  }
+
+  &__reviws {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-count {
+      line-height: 14px;
+      margin: 0;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: auto 0 16px;
+
+    &-how {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $text--color--one;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 11px 0;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    border-radius: 5px;
+    border: 1px solid $buy--color;
+    color: $header--bcg--color;
+    background-color: $buy--color;
+
+    &:hover {
+      color: $buy--color;
+      background-color: $header--bcg--color;
+      transition: 300ms;
+    }
+  }
+
+  &__price-lable {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text--color--two;
+    margin: 0;
+  }
+
+  &:hover .card__title {
+    color: $card__hover;
+  }
 }
```

Comment:
The last file is for the card block, so there should be just imports in the main.scss